payloads/v10/channel.ts

```diff
@@ -688,6 +689,14 @@ export enum MessageType {
 	GuildInviteReminder,
 	ContextMenuCommand,
 	AutoModerationAction,
+
+	InteractionPremiumUpsell = 26,
```

Comment:
Shouldn't we have `RoleSubscriptionPurchase` (25) above this one? That way we don't need to specify it. 👀 

I don't see any open PR adding that type.

Reply:
#665